auto_gptq/modeling/llama.py

```diff
@@ -24,8 +26,61 @@ class LlamaGPTQForCausalLM(BaseGPTQForCausalLM):
         ["mlp.down_proj"]
     ]
 
-    fused_attn_module_type = FusedLlamaAttentionForQuantizedModel
-    fused_mlp_module_type = FusedLlamaMLPForQuantizedModel
+    @staticmethod
+    def _fuse_attention(
+        model: PreTrainedModel,
+        attn_op: Optional[AttentionOp] = None,
+        trainable: bool = False
+    ) -> None:
+        model_config = model.config
+        num_heads = model_config.num_attention_heads
+        scale = (model_config.hidden_size // num_heads) ** -0.5
+        layers = model.model.layers
+
+        rope_cache = build_rope_cache(
```

Comment:
I see baichuan7b is the rope and baichuan13b is the alibi?

Reply:
Thank you for point this! Will check again and update the code if necessary